### DISCUSSION TITLE
Incorrect 3D Embedding for Quercetin dihydrate

### DISCUSSION THREAD

**Discussion by MagicPF** [rdkit/rdkit#4799] [Q&A]
smiles = "C1=CC(=C(C=C1C2=C(C(=O)C3=C(C=C(C=C3O2)O)O)O)O)O.O.O"
mol = Chem.MolFromSmiles(smiles)
AllChem.Compute2DCoords(mol)
m3 = Chem.AddHs(m2)
AllChem.EmbedMolecule(m3)
pos = Chem.MolToMolBlock(m3)

According to the visualization result based on pos, the structure was not respect to the real structure, may I ask for the reason and solution?

**Comment by greglandrum** [comment]
@MagicPF, molecules have multiple possible 3D conformers available to them. Why do you think the one that the RDKit provided is incorrect?
